src/components/calcite-accordion-item/calcite-accordion-item.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+describe("calcite-accordion-item", () => {
+  it("renders", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent("<calcite-accordion-item></calcite-accordion-item>");
+    const element = await page.find("calcite-accordion-item");
+    expect(element).toHaveClass("hydrated");
+  });
+});
```

Comment:
Given that the rest of the project has been pretty well-tested, I think it would be nice to have a few more tests here.

Reply:
yeah could probably just test that it's emitting the right events based on user input or something

Reply:
Yeah I've been wondering about best practices there, the parent-level tests touch on what would go here...

Reply:
Updated tests on `calcite-accordion`, which test `calcite-accordion-item`